### PR TITLE
Update updatecli configuration to be compliant with version 0.2.0

### PR DESCRIPTION
**Pull request by olblak**
Signed-off-by: Olivier Vernin <olivier@vernin.me>
Requires v0.2.0 to be defined in the PodTemplate #998 

- Use the new helmchart resource to automatically handle helm chart like updating appVersion, chart version or deleting requirements.lock when needed.
- Remove several deprecated fields and rely more on default values.
- Specify a `pipelineID` for every pipeline so we are certain to always use the same git branch when we open a pull request
- Switch to the new default multi sources syntax

### CODE REVIEW

File: updateCli/updateCli.d/charts/falco.tpl [hunk 1]
```diff
@@ -1,9 +1,11 @@
-source:
-  kind: helmChart
-  spec:
-    url: https://falcosecurity.github.io/charts
-    name: falco
-
+title: Bump falco helm chart version
+pipelineID bumpfalcohelmchart
```
Comment (timja) [inline updateCli/updateCli.d/charts/falco.tpl:2]:
```suggestion
pipelineID: bumpfalcohelmchart
```

Comment (olblak) [reply updateCli/updateCli.d/charts/falco.tpl:2]: Thanks, I have to double check why I didn't catch that :/

File: updateCli/updateCli.d/updateCliPodTemplate.tpl [hunk 2]
```diff
@@ -39,7 +41,8 @@ targets:
   imageTag:
     name: "Update Updatecli verison in PodTemplates.yaml"
```
Comment (olblak) [inline updateCli/updateCli.d/updateCliPodTemplate.tpl:42]:
```suggestion
    name: "Update Updatecli version in PodTemplates.yaml"
```

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by olblak** [comment]
I'll merge it on Monday once the update-center certificate is rotate so I don't get distracted by this pull request

**Pull request merged**